docs/docs/development/hardware-integration/dongle.mdx

```diff
@@ -0,0 +1,301 @@
+---
+title: Keyboard Dongle
+sidebar_label: Keyboard Dongle
+---
+
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+
+A bluetooth dongle can be added to any keyboard running ZMK. The result is a [split keyboard](../../features/split-keyboards.md) with the dongle as ["central"](../../features/split-keyboards.md#central-and-peripheral-roles). There are a number of advantages to adding a dongle, but also some disadvantages:
+
+Benefits:
+
+- For split keyboards, having the dongle for the "central" role results in [improved battery life](/power-profiler) for the former central part, as it is now a peripheral.
+- It is easier to connect to the host device since dongles are typically connected via USB.
+
+Disadvantages:
+
+- An extra [board](../../faq.md#what-is-a-board) is needed (any BLE-capable board will work).
+- The keyboard becomes unusable without the dongle.
+
+Depending on how the dongle is used, there are some additional [latency considerations](../../features/split-keyboards.md#latency-considerations) to keep in mind.
+The addition of the dongle adds an extra "hop" for the former central, increasing its latency to that of a peripheral. The other parts are unchanged latency-wise. There is also a commonly occurring case where the peripherals benefit.
+Assuming the dongle is connected to USB and the former central would have been connected via bluetooth to the host if the dongle wasn't present:
+
+- The former central will have its latency increased by about 1ms from the extra USB "hop"
+- The other parts will have their average latency _decreased_ by 6.5ms from the replacement of a BLE "hop" with a "USB" hop.
+
+As a result, for this common use case the average latency of the keyboard decreases.
+
+## Adding a Dongle
+
+The approach taken to adding a dongle is _generally_ always the same[^1].
+Whether your keyboard consists of a board and a shield or is just a board is irrelevant.
+
+To add a dongle, you will create a simplified form of a [new shield](new-shield.mdx).
+The approach described below assumes the dongle will not have any keys assigned to itself, so it will not work for that scenario. If you want to understand the details of how it all works better, please read through the [new shield guide](new-shield.mdx).
```

Comment:
```suggestion
The approach described below assumes the dongle will not have any keys assigned to itself, so it will not work for that scenario.
If you want to understand the details of how it all works better, please read through the [new shield guide](new-shield.mdx).
```
For consistency